package.json

```diff
@@ -16,13 +19,20 @@
     "mongoose": "^5.7.3",
     "mysql": "^2.17.1",
     "nodemon": "^1.19.3",
+    "path": "^0.12.7",
```

Comment:
I believe path is build into node.js. Is this a different path node packet?

Reply:
so it is... neat!